setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
+exclude = client/__init__.py
```

Comment:
I would just delete init if that's the case, cause then we have to remember to remove exclude
can also just put 
pass